supabase/migrations/20240405104731_add_audit_log_table.sql

```diff
@@ -0,0 +1,118 @@
+create table "public"."audit_log" (
+    "primary_key" uuid not null default gen_random_uuid(),
+    "user_id" uuid not null,
+    "action" text,
+    "client_id" uuid,
+    "collection_centre_id" uuid,
+    "event_id" uuid,
+    "family_member_id" uuid,
+    "list_id" uuid,
+    "list_hotel_id" uuid,
+    "packing_slot_id" uuid,
+    "parcel_id" uuid,
+    "status_order" text,
+    "website_data" text,
```

Comment:
status_order and website_date do not have primary keys

Reply:
they actually have primary keys! They just happen to be text, not uuid or number! so this is fine!